src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
That works. 
I understand your guys' point, but having to come up with a name when not needed is probably less of a burden than trying to roll keys over or make sure you're removing the right key when revoking a consumer's key when the list grows beyond just a couple of them.
having a:
``` json
"masterKey": {
"consumerAKey" : "...",
"consumerBKey" : "...",
"consumerZKey" : "..."
}
```
Is a lot easier to manage